core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/DatabaseService.java

```diff
@@ -26,6 +26,11 @@
 public abstract class DatabaseService<T extends Database> implements IDatabaseService<T> {
 
     public Connection getConnection(T database) throws SQLException {
+        //Avoid creating a Hikari Connection pool for Bigquery
+        if (database.getDatabaseConnection().getType().equalsIgnoreCase("bigquery")) {
+            return DatabaseConnectionHandler.getInstance().getConnection(database.getDatabaseConnection());
+        }
+
```

Comment:
Method 'public Connection getConnection(T database) throws SQLException' should be overriden in 'core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/bigquery/BigqueryDatabaseService.java'. See 'io.metadew.iesi.connection.database.sqlite.SqliteDatabaseService' for example

Reply:
Resolved in next commit